enterprise/admin/views.py

```diff
@@ -767,14 +870,16 @@ def get(self, request, customer_uuid):
         Returns:
             django.http.response.HttpResponse: HttpResponse
         """
-        context = self._build_context(request, customer_uuid)
-        manage_learners_form = ManageLearnersForm(
-            user=request.user,
-            enterprise_customer=context[self.ContextParameters.ENTERPRISE_CUSTOMER]
-        )
-        context.update({self.ContextParameters.MANAGE_LEARNERS_FORM: manage_learners_form})
+        enterprise_customer = EnterpriseCustomer.objects.get(uuid=customer_uuid)  # pylint: disable=no-member
```

Comment:
are we sure this will never raise `EnterpriseCustomer.DoesNotExist` error?

Reply:
Yes.